modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsEpics.scala

```diff
@@ -422,9 +420,9 @@ final class TcsEpics[F[_]: Sync](epicsService: CaService, tops: Map[String, Stri
    * an error. A better solution is to detect the edge, from not in position to in-position.
    */
   private val AGSettleTime = 1100.milliseconds
-  def waitAGInPosition(timeout: Time): SeqAction[Unit] = SeqAction(Thread.sleep(AGSettleTime.toMilliseconds.toLong)) *>
-    SeqAction(filteredAGInPositionAttr.reset).flatMap(
-      waitForValue[java.lang.Double](_, 1.0, timeout, "AG inposition flag"))
+  def waitAGInPosition(timeout: Time): F[Unit] = Sync[F].delay(Thread.sleep(AGSettleTime.toMilliseconds.toLong)) *>
```

Comment:
It maybe better to use `Timer[F].sleep(AGSettleTime.milliseconds)` for all delays rather than making the `Thread` sleep